examples/typescript/asset_uploader.ts

```diff
@@ -13,27 +41,56 @@ const example = async () => {
   await aptos.fundAccount({ accountAddress: account.accountAddress, amount: 1_000_000 });
 
   // Fund a node
-  const fundResponse = await assetUploader.fundNode({ account, amount: 1000 });
+  const fundResponse = await assetUploader.fundNode({ account, amount: 1_000_000 });
   console.log("fund response", fundResponse);
 
-  // Upload data
-  const uploadDataResponse = await assetUploader.uploadData({ account, data: "Hello Aptos" });
-  console.log("upload response", uploadDataResponse);
-
-  // Upload file
+  // Upload collection media
+  const collectionMediaPath = "/collection_media_file_path"; // Update this path
   const uploadFileResponse = await assetUploader.uploadFile({
     account,
-    file: "/Users/maayansavir/Desktop/pfp.jpeg",
+    file: collectionMediaPath,
+  });
+  const collectionMediaManifestID = uploadFileResponse.id;
+  const collectionMediaURI = `https://arweave.net/${collectionMediaManifestID}`;
+  console.log("Collection Media URL:", collectionMediaURI);
+
+  // Update collection metadata JSON file
+  const collectionMetadataJsonPath = "/collection_metadata_json_file_path"; // Update this path
+  updateImageField(collectionMetadataJsonPath, collectionMediaURI);
+
+  // Upload updated collection metadata json
+  const uploadJsonResponse = await assetUploader.uploadFile({
+    account,
+    file: collectionMetadataJsonPath,
   });
-  console.log("uploadFileResponse", uploadFileResponse);
+  const collectionMetadataJsonManifestID = uploadJsonResponse.id;
+  const collectionMetadataJsonURI = `https://arweave.net/${collectionMetadataJsonManifestID}`;
+  console.log("Collection metadata json URL:", collectionMetadataJsonURI);
 
-  // Upload folder
+  // Upload token media folder
+  const tokenMediaFolderPath = "/token_media_folder_path"; // Update this path
   const uploadFolderResponse = await assetUploader.uploadFolder({
     account,
-    folder: "/Users/maayansavir/Desktop/test-apps/web-app",
+    folder: tokenMediaFolderPath,
+  });
+  const tokenMediaFolderManifestID = uploadFolderResponse!.id;
+  const tokenMediaFolderURI = `https://arweave.net/${tokenMediaFolderManifestID}`;
+  console.log("Token media URL folder:", tokenMediaFolderURI);
+
+  // Update token metadata JSON files with token media URL
+  const tokenMetadataJsonFolderPath = "/token_metadata_json_folder_path"; // Update this path
+  updateImageFieldsInFolder(tokenMetadataJsonFolderPath, tokenMediaFolderURI);
+
+  // Upload the updated token metadata json folder
+  const uploadMetadataFolderResponse = await assetUploader.uploadFolder({
+    account,
+    folder: tokenMetadataJsonFolderPath,
   });
-  console.log("uploadFolderResponse", uploadFolderResponse);
-  process.exit(0);
+  const tokenMetadataJsonManifestID = uploadMetadataFolderResponse!.id;
+  const tokenMetadataJsonFolderURI = `https://arweave.net/${tokenMetadataJsonManifestID}`;
```

Comment:
ideally, that should come from Irys response (I brought it up with the Irys team) but until then maybe we can add it to the response from irys.uploadFolder (dont need to be part of this PR)